restcomm/restcomm.docs/sources-asciidoc/src/main/asciidoc/configuration/asr/asr.adoc

```diff
@@ -0,0 +1,13 @@
+[[intro]]
+= Automatic Speech Recognition Integration
+
+In order to enable ASR functionality an appropriate configuration has to be added. There are list of available drivers supported by mediaserver and a default one, which can be used by Restcomm. These are environment variables you need to set in restcomm.conf file
+
```

Comment:
@deruelle it's configuration for restcomm. this list will be displayed on the account page